packages/types/src/tone.types.ts

```diff
@@ -0,0 +1,15 @@
+export type Tone = {
+  id: string;
+  name: string;
+  promptTemplate: string;
+  isSystem: boolean;
+  createdAt: number;
+  sortOrder: number;
+};
```

Comment:
It'd be handy to have these on the cloud so they can be shared between devices, we should throw in a firebase endpoint